src/services/data/types.ts

```diff
@@ -769,8 +769,8 @@ export interface DataDocumentsService {
   unlinkGroups(
     schemaIdOrName: ObjectId | string,
     documentId: ObjectId,
-    requestBody: {
```

Comment:
In my opinion it is safer and better ux to keep both required and create an extra `unlinkAllGroups` function.

But that is more because I don't really like the design of the end point as it is than due to this fix.

Reply:
I agree! But since it is describing the endpoint I feel this should exist (at least) in this form. Would it make sense to add a ticket to create the extra helper at some point?

Reply:
Ahh okay, I didn't fully understand you position. I agree keeping the need for an empty object helps to prevent you shooting yourself in the foot. Thomas also mentioned this in his original issue (#698). Fixed